demo/src/main/resources/oiosaml.properties

```diff
@@ -39,3 +39,8 @@ oiosaml.servlet.secondary.page.error=error.jsp
 oiosaml.servlet.secondary.page.logout=index.jsp
 oiosaml.servlet.secondary.page.login=index.jsp
 
+oiosaml.servlet.audit.logger.classname=dk.gov.oio.saml.audit.Slf4JAuditLogger
```

Comment:
I'm not sure that it makes sense to add default values as configuration here. Only what is strictly necessary should be present here - otherwise it will just confuse.